be/src/runtime/exec_env.cpp

```diff
@@ -503,6 +503,9 @@ void ExecEnv::_destroy() {
         delete _compaction_mem_tracker;
         _compaction_mem_tracker = nullptr;
     }
+
+    _lake_tablet_manager->prune_metacache();
```

Comment:
I suggest prune cache in _lake_tablet_manager deconstructor

Reply:
If prune cache is placed in  destructor, it will cause illegal memory access. _tablet_meta_mem_tracker is destroyed before _lake_tablet_manager. But tablet_schema is being traced by _tablet_meta_mem_tracker.

Reply:
run starrocks_clang-format